python/lsst/analysis/ap/plotImageSubtractionCutouts.py

```diff
@@ -50,6 +49,44 @@
 from . import apdb
 
 
+class ButlerCache:
```

Comment:
Do you really want this exposed outside? maybe consider adding an `__all__` even just for documentation reasons.

Reply:
This module already had an `__all__`. I've also renamed the class to `_ButlerCache` to emphasize the private-ness of it.